library/command/command/build.gradle

```diff
@@ -10,4 +10,12 @@ qslModule {
 			"qsl_base",
 			"resource_loader"
 	])
+	coreTestmodDependencies([
+	        "networking"
+	])
+}
+
+dependencies {
+	// OPTIONAL dependency on networking
+	compileOnly project(path: ":core:networking", configuration: "dev")
```

Comment:
CC @QuiltMC/qsl-core-library should we use compile only or just impl? i can update #72 to reflect the decision here